jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-6a.yaml

```diff
@@ -96,6 +96,14 @@
           default: services=2
           description: HA configuration for clusters. Make sense only if hacloud=1
       - string:
+          name: networkingplugin
+          default: openvswitch
+          description: networking plugin to be used by neutron
+      - string:
```

Comment:
Hi, could you add spaces between -string sections like in the beginning of the scenario (and in others scenarios as well) so it will look holistically.